### PR TITLE
Describe ENV vars in the .env.example

### PR DESCRIPTION
**Pull request by ruslankhaertdinov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,13 @@ It's based on Rails 4 and Ruby 2.2.2.
 * `bin/ci` - should be used in the CI or locally
 * `bin/server` - to run server locally
 
+## ENV vars
```
Comment (vast) [inline README.md]: why not put this comments closer to the described things, right in `.env.example`?

Comment (ruslankhaertdinov) [inline README.md]: Good point, thanks.

**Comment by ruslankhaertdinov** [comment]
@vast done

**Comment by timurvafin** [comment]
@ruslankhaertdinov could you please add comments above each env var

```ruby
# comment 1
VARIABLE_1

# comment 2
VARIABLE_2
```

**Comment by ruslankhaertdinov** [comment]
@timurvafin done

**Comment by timurvafin** [comment]
@ruslankhaertdinov please cleanup commits and we will be ready to merge.

**Comment by ruslankhaertdinov** [comment]
@timurvafin done

**Pull request merged**